### PR TITLE
[Function][Admin] Create Add Category function

### PR DESCRIPTION
**Pull request by kaz-m-93**
i created function for add category page
asana:
https://app.asana.com/0/1199094899705013/1199181177413496

### CODE REVIEW

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -5,4 +5,20 @@ def index
   def new
     @category = Category.new
   end
+
+  def create
+    @category = Category.new(category_params)
+    if @category.save
+      flash[:success] = "a category has been created"
+      redirect_to admin_categories_path
```
Comment (CharlotteDS) [inline app/controllers/admin/categories_controller.rb:13]: Please use `_url` for controllers and `_path` for views.
So for this, you should use `admin_categories_url`

**Comment by kaz-m-93** [comment]
fix all _path to _url and add category page a little bit

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <% provide(:title, "login") %>
 <div class="static-page-form">
-    <%= form_for(:session, url: login_path) do |f| %>
+    <%= form_for(:session, url: login_url) do |f| %>
```
Comment (CharlotteDS) [inline app/views/sessions/new.html.erb:3]: Please use `_path` inside forms.

File: app/views/users/_form.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<%= form_for(@user, url: signup_path) do |f| %>
+<%= form_for(@user, url: signup_url) do |f| %>
```
Comment (CharlotteDS) [inline app/views/users/_form.html.erb:1]: Please use `_path` inside forms.

**Comment by kaz-m-93** [comment]
fixed _url to _path in form_for

**Review by CharlotteDS — APPROVED** [review]

**Pull request merged**